### PR TITLE
JOSS Paper content

### PR DESCRIPTION
**Pull request by j-i-l**
The branch `JOSSpaper_noT` gathers all paper specific content, including `paper.md`, `paper.bib`, etc.
Merge in main before submission. Do revisions during submission process on this branch then again.

Content that is relevant for abn package also outside the scope of the paper, but that was added/adapted/edited during the preparation of the paper should not reside in this branch.

### CODE REVIEW

**Comment by j-i-l** [comment]
@matteodelucchi https://github.com/furrer-lab/abn/pull/37/commits/9d4d8b9b18e8550ff5568214ec77fec40083dc2c is just a suggestion, feel free to revert or adapt :)

**Comment by j-i-l** [comment]
@matteodelucchi I've also added a comment to https://github.com/orgs/furrer-lab/projects/1?pane=issue&itemId=57489555

**Comment by matteodelucchi** [comment]
> @matteodelucchi I've also added a comment to https://github.com/orgs/furrer-lab/projects/1?pane=issue&itemId=57489555

thanks, reads better without the repetition.

File: vignettes/paper.Rmd
Comment (matteodelucchi) [inline vignettes/paper.Rmd:1]: we can remove this, right?

**Review by matteodelucchi — APPROVED** [review]

File: vignettes/paper.md [hunk 1]
```diff
@@ -37,20 +37,24 @@ BNs are a type of statistical model that leverages the principles of Bayesian st
 ABN models extend the concept of generalized linear models, typically used for predicting a single outcome, to scenarios with multiple dependent variables (e.g. @kratzer_additive_2023).
 This makes them a powerful tool for understanding complex, multivariate datasets.
 
+[//]: # (You switch from BN to ABN 1st paragraph, then revisit BN and again to ABN. First all BN then To ABN.)
```
Comment (matteodelucchi) [inline vignettes/paper.md:40]: Thanks, will improve accordingly.

File: vignettes/paper.md [hunk 2]
```diff
@@ -37,20 +37,24 @@ BNs are a type of statistical model that leverages the principles of Bayesian st
...
 This limitation can pose challenges when dealing with continuous or mixed-type data (i.e., data that includes both continuous and discrete variables) or when attempting to model complex relationships that do not fit these restricted categories.
 For a comprehensive overview of structure learning algorithms, including those applicable to mixed-type data, we refer the reader to the works of @kitson_survey_2023 and @zanga_survey_2022.
 In the context of patient data, the study from @delucchi_bayesian_2022 has discussed further details and strategies for handling this scenario, particularly in relation to the `abn` package and the widely used `bnlearn` package [@bnlearn2010].
 
+[//]: # (Next line you talk about `these limitations`. Suboptimal, ascendant not clear. Especially in the previous paragraph you only talk about `this limitation`.)
```
Comment (matteodelucchi) [inline vignettes/paper.md:52]: thanks!

File: vignettes/paper.md [hunk 3]
```diff
@@ -69,7 +73,9 @@ Its unique contribution is the implementation of mixed-effect BN models, thereby
 # Implementation
 As outlined in @kratzer_additive_2023, the package's comprehensive framework integrates the mixed-effects model for clustered data, considering data heterogeneity and grouping effects.
 However, this was confined to a Bayesian context. 
-With the release of `abn` major version 3 this was completed with an implementation under the informaiton-theoretic ("mle") setting.
+With the release of `abn` major version 3, this was completed with an implementation under the information-theoretic ("mle") setting.
+
+[//]: # (Maybe frame in line 74: more as there existed a first attempt towards mixed-effects in a Bayesian fitting context. or even a bit weaker.)
```
Comment (matteodelucchi) [inline vignettes/paper.md:78]: Good comment! Will change the first sentence of this section accordingly.

File: vignettes/paper.md [hunk 4]
```diff
@@ -83,6 +89,8 @@ Currently, only single-layer clustering is supported (e.g., for `method = "mle"`
 With a Bayesian approach (`method = "bayes"`), `abn` utilizes its own implementation of the Laplace approximation as well as the `INLA` package [@inla2013] to fit a single-level hierarchical model for Binomial, Poisson, and Gaussian distributed variables. 
 Independent of the type of data, multinomial distributed variables are not yet implemented with `method ="bayes"` (details in the [online manual](https://r-bayesian-networks.org/quick_start_example.html)).
 
+[//]: # (Line 90 needed?)
```
Comment (matteodelucchi) [inline vignettes/paper.md:92]: We can remove it in favor of the word count. It is mentioned in more detail on the homepage anyway.

File: vignettes/paper.md [hunk 5]
```diff
@@ -109,13 +117,16 @@ The development version of the `abn` package is hosted on [GitHub](https://githu
 devtools::install_github("furrer-lab/abn")
 ```
 
+[//]: # (Comment should be above, do you mention that this is all under `R`??)
```
Comment (matteodelucchi) [inline vignettes/paper.md:120]: Will make this explicit in the first sentence. Thanks!

**Review by matteodelucchi — CHANGES_REQUESTED** [review]

**Pull request merged**